tests/test_denormalized.c

```diff
@@ -0,0 +1,11 @@
+#include <float.h>
+
+void __initalize_cop1();
+
+void test_denormalized(TestContext *ctx) {
+    /* Create a volatile float, so gcc does not optimize it out */
+    volatile float x = 1.0f;
+    x /= FLT_MAX;
+
+    ASSERT(x == 0.0f, "Denormalized float was not flushed to zero");
```

Comment:
Also a comment here to explain that we're testing that an exception is not generated when a denormal is created.

(Technically, a test that could fail with an exception should trap it to avoid trashing the whole testsuite when it fails, but that's something we don't have helpers for, so don't worry. I'm sure it would be a new rabbit hole...)